### ISSUE TITLE
Agent reporting non-existing jake capability

### ISSUE THREAD

**Issue by ppanyukov** [Microsoft/vso-agent#201]
Just noticed that the agent reports `jake` as capability even if it's entirely missing.

This line: https://github.com/Microsoft/vso-agent/blob/master/src/agent/environment.ts#L194

```
    resolveCapability(filteredEnv, 'jake', null, '.'); // If not in global path, use jake packaged in this agent
```

assumes it's bundled with the agent but I can't it anywhere.

Am I missing something?

**Comment by bryanmacfarlane** [comment]
Good catch.  I'll remove.

**Comment by bryanmacfarlane** [comment]
Fixed with commit 024aa6f4c8ea8674993173c6763cd73a98c4171c

**Comment by bryanmacfarlane** [comment]
will be in next 0.5 drop
